hedera-mirror-common/src/test/java/com/hedera/mirror/common/domain/DomainBuilder.java

```diff
@@ -1118,7 +1123,11 @@ public byte[] bytes(int length) {
 
     public EntityId entityId() {
         long nextNum = number() + LAST_RESERVED_ID;
-        return EntityId.of(commonProperties.getShard(), commonProperties.getRealm(), nextNum);
+        return entityId(nextNum);
+    }
+
+    public EntityId entityId(long num) {
```

Comment:
To disambiguate with db encoded long.

```suggestion
    public EntityId entityNum(long num) {
```